src/classes.py

```diff
@@ -2,20 +2,47 @@
 # constructor
 
 # YOUR CODE HERE
-        
+class LatLon:
+    def __init__(self, lat, lon):
+        self.lat = lat
+        self.lon = lon
+
+
 # Make a class Waypoint that can be passed parameters `name`, `lat`, and `lon` to the
 # constructor. It should inherit from LatLon. Look up the `super` method.
 
 # YOUR CODE HERE
+class Waypoint(LatLon):
+    def __init__(self, name, lat, lon):
+        LatLon.__init__(self, lat, lon)
+        self.name = name
+
+    def __str__(self):
+        return ''
+
 
 # Make a class Geocache that can be passed parameters `name`, `difficulty`,
 # `size`, `lat`, and `lon` to the constructor. What should it inherit from?
 
 # YOUR CODE HERE
+class Geocache(Waypoint):
+    def __init__(self, name, difficulty, size, lat, lon):
+        Waypoint.__init__(self, name, lat, lon)
```

Comment:
super could be applied here to make your code DRYer in the long run
super().__init__(name, lat, lon)

over all nice job finishing classes, this part will help you w/the rest of the week